services/wallet/collectibles/service.go

```diff
@@ -61,6 +61,11 @@ const (
 	FetchTypeFetchIfCacheOld
 )
 
+type TxHashData struct {
+	Hash common.Hash
```

Comment:
hmm since I only needed the timestamp originally it was enough for me to get the real Tx Hash, but it might've made more sense to lookup the specific row (TxID) instead